package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "ncc build src/index.js",
     "lint": "eslint .",
-    "test": "tap",
+    "test": "tap test/**.test.js",
```

Comment:
By default it uses `test/*.js` which includes [this file](https://github.com/fastify/github-action-merge-dependabot/blob/main/test/moduleChanges.js) which prints a warning because it doesn't have any actual tests on it.